tests/test_api.py

```diff
@@ -58,3 +66,7 @@ def test_sms_render_person(self):
             home_state='California',
             entry_date=datetime.datetime(2010, 1, 1))
         assert handler.render_person(person) == 'John Smith / From: California'
+
+
+if __name__ == '__main__':
```

Comment:
You can run a specific test by e.g.,

    $ tools/unit_tests -k APITests

So I believe you don't need to add this.